test/DocumentFormat.OpenXml.Tests/Common/CultureHelper.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DocumentFormat.OpenXml.Tests.Common
+{
+    internal readonly struct CultureInfoTester : IDisposable
+    {
+        private readonly CultureInfo _old;
+
+        public CultureInfoTester(CultureInfo desired)
```

Comment:
You could provide a string overload that creates a cultureinfo to minimize `new CultureInfo(...)` at the call site

Reply:
Done